editor/editor_node.h

```diff
@@ -773,8 +773,8 @@ class EditorNode : public Node {
 	Ref<Theme> get_editor_theme() const { return theme; }
 	Ref<Script> get_object_custom_type_base(const Object *p_object) const;
 	StringName get_object_custom_type_name(const Object *p_object) const;
-	Ref<Texture2D> get_object_icon(const Object *p_object, const String &p_fallback = "Object") const;
-	Ref<Texture2D> get_class_icon(const String &p_class, const String &p_fallback = "Object") const;
```

Comment:
What is the purpose of removing `const`? Do these methods need to modify the EditorNode instance?

Reply:
They use resource loader load which isn't const IIRC, but I might be able to add it back since previously I used a manual cache system.